.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 install:
 - echo "install nothing!"
 script:
-- echo "no tests!"
+- echo "Checking JavaScript is 'Standard'"
+- npm install -g standard
```

Comment:
Installing packages should not happen in the testing phase. Also, you will not have permission to install a package globally using `-g` on a Travis build. You should do the following instead:

1. On your local machine, `cd` to the `mobile` directory
2. Run `npm install --save-dev standard
3. In the `package.json`, prepend the `"test"` script with `standard && ...`
4. Commit all changes to `package.json`
5. In the `.travis.yml` file, change the `script` part to this:
```yml
script:
- npm test
```
6. Change the `install` section to:
```yml
install:
- npm install
```

Reply:
Thanks for the review — I've implemented these changes in the most recent commit.

Success! Travis fails if `standard` reports errors (see screenshot)
<img width="1070" alt="screen shot 2018-04-24 at 13 27 28" src="https://user-images.githubusercontent.com/17081120/39186733-54061802-47c3-11e8-888a-e53360998c39.png">